Utils/distroutils.py

```diff
@@ -263,6 +263,21 @@ def __init__(self, config):
         self.sudoers_dir_base = '/usr/local/etc'
         self.distro_name = 'Ubuntu'
 
+    def restart_ssh_service(self):
```

Comment:
Was this added by mistake? I don't see where it is called.

Reply:
No, it was added on purpose, on newer Ubuntu distros, while the operation succeeds, the password doesn't reset. Function is being called here: https://github.com/Azure/azure-linux-extensions/blob/adb7396fb0479db542973917edb6fc709de9851c/VMAccess/vmaccess.py#L173C1-L174C1